### ISSUE TITLE
Misinformation in the website

### ISSUE THREAD

**Issue by skill-issue-lmao** [Endermanch/Endermanch#1]
![image](https://github.com/Endermanch/Endermanch/assets/146542735/74f76970-5593-4637-bbc4-3f51dc28d45b)
w*ndows is not the best. massive skill issue

**Comment by Elefant9477** [comment]
can i just.. see the part above?

**Comment by skill-issue-lmao** [comment]
Project computer section at https://malwarewatch.org/about.html

**Comment by Elefant9477** [comment]
I agree that Windows isn't the best (Win 11 sucks).
By the way at last (maybe) for him, his malware runs best on Windows.

**Comment by skill-issue-lmao** [comment]
Yeah i know there is no funny malware on Linux but he is already running malware on vm's. So thats not the reason.
